### ISSUE TITLE
TypeError: Couldn't find jasmine environment

### ISSUE THREAD

**Issue by guyellis** [gribnoysup/setup-polly-jest#10]
Getting:

`TypeError: Couldn't find jasmine environment. Make sure that you are using "setupJasmine" in jasmine/jest environment or that you provided proper jasmine environment when calling "setupJasmine"`

I can see where in the setup-polly-jest code this is happening:

```javascript
function setupJasmine(Polly, defaults = {}, ctx = global) {
  if (!ctx.jasmine || ctx.jasmine && typeof ctx.jasmine.getEnv !== 'function') {
    throw new TypeError('Couldn\'t find jasmine environment. Make sure that you are using "setupJasmine" in ' + 'jasmine/jest environment or that you provided proper jasmine environment when calling "setupJasmine"');
  }
```

It doesn't look like the `setupJasmine` method is intended for external consumption as it's wrapped by `setupPolly` - I'm guessing that the `ctx` param is there for testing.

So in my case the `global` doesn't have a `jasmine` object - visible when debugging. What have I not setup to cause this to happen?

Versions:

```
    "setup-polly-jest": "0.6.0",
    "@pollyjs/adapter-node-http": "4.0.3",
    "@pollyjs/core": "4.0.2",
    "@pollyjs/persister-fs": "4.0.2",
    "jest": "25.1.0",

NodeJS 12.14.1
```

**Comment by gribnoysup** [comment]
Hey @guyellis! I guess as mentioned in the [caveats](https://github.com/gribnoysup/setup-polly-jest#caveats) section some major changes in freshly released jest 25 broke the code of this library.

I'll need some time to look into it and figure out what's going on there. For now I guess the only option is to roll back to jest 24 if you want to use the library or use polly directly without the helper

**Comment by gribnoysup** [comment]
Hmm... do you mind creating a reproduction example or sharing your full jest config?

I just tried it with the latest versions of jest and polly and everything seems to work fine. Are you sure you are running jest with [jasmine set as a test runner](https://jestjs.io/docs/en/configuration#testrunner-string) (it should be by default)?

**Comment by guyellis** [comment]
@gribnoysup I am using:

```
  testRunner: 'jest-circus/runner',
```

So I'm guessing that's the issue. I didn't realize that Jasmine needs to be the test runner. I ran into a known issue with Jasmine where it "fails silently on afterAll within a describe block" which is why I switched to jest-circus. That was a while back. Let me see if I'm able to switch back to Jasmine.

How difficult would it be for `setup-polly-jest` to use `jest-circus`? Or is this something that PollyJS does not support?

**Comment by gribnoysup** [comment]
It's on my TODO list for some time already, but I just didn't have time to get to it yet. I guess I'll open an issue for `circus` support for visibility and will probably change the error message for now to something that mentions `jest/jasmine2` specifically, not just jest

The silent thing should be fixed in the latest released version hopefully after #6 was merged. Please feel free to open a new issue for this one if it's still happening for you

**Comment by guyellis** [comment]
Perhaps a question in that error message like "Do you have something other than Jasmine2 set against testRunner in your jest.config.js file?"

I know that gets wordy but might help divert this sort of question.

Thank you for creating this module and supporting it! You are awesome! :trophy:

**Comment by gribnoysup** [comment]
Yeah, good idea! I'll probably include a mention of `testRunner` config option in the message. I think wordy is really not that bad in this case, better to be verbose 👍

**Comment by gribnoysup** [comment]
Hey @guyellis! I just added [support for `jest-circus`](https://github.com/gribnoysup/setup-polly-jest#supported-test-runners) in the latest release if you want to try it out

**Comment by guyellis** [comment]
I'll try it out. Thanks @gribnoysup
